app/ts/App.tsx

```diff
@@ -289,13 +295,20 @@ export function App() {
 		await refresh(maybeReadClient.deepValue, maybeWriteClient.deepValue, isDeployed.deepValue, chainId.value).catch(handleUnexpectedError)
 	}
 
-	const buttonMicahWithdraw = async  () => {
+	const buttonMicahWithdraw = async () => {
 		if (maybeWriteClient.deepValue === undefined) throw new Error('wallet not connected')
 		if (isMicah.value !== true) throw new Error('you are not micah!')
 		await micahWithdraw(maybeWriteClient.deepValue).catch(handleUnexpectedError)
 		await refresh(maybeReadClient.deepValue, maybeWriteClient.deepValue, isDeployed.deepValue, chainId.value).catch(handleUnexpectedError)
 	}
 
+	const buttonMicahSetWithdrawsEnabled = async () => {
+		if (maybeWriteClient.deepValue === undefined) throw new Error('wallet not connected')
+		if (isMicah.value !== true) throw new Error('you are not micah!')
```

Comment:
These errors are unhandled.  If the code is somehow reached, then they'll show up in logs and nowhere else (and even showing up in logs is dependent on the runtime implementation details).